downstream/modules/platform/con-aap-example-architecture.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: CONCEPT
+// This module is included in assembly-aap-architecture.adoc
+[id='aap_example_architecture_{context}']
+= Example {PlatformNameShort} architecture
```

Comment:
The anchor isn't highlighting in blue like it ought to--is the italicization throwing off the formatting?